includes/ingest.form.inc

```diff
@@ -70,16 +71,32 @@ function islandora_basic_collection_get_ingest_configuration(AbstractObject $col
  *   The Drupal form state.
  * @param array $models
  *   The models to include in the form.
+ * @param string $parent_pid
+ *   The pid of parent collection.
  *
  * @return array
  *   The Drupal form definition.
  */
-function islandora_basic_collection_select_content_model_form(array $form, array &$form_state, array $models) {
+function islandora_basic_collection_select_content_model_form(array $form, array &$form_state, array $models, $parent_pid = NULL) {
```

Comment:
Again, not necessary to pass along the `$parent_pid`... It's available in the shared storage in the `$form_state` (use `islandora_ingest_form_get_shared_storage()` to access it).